static/src/uu/variables.scss

```diff
@@ -5,20 +5,16 @@ $black-alt: #333;
 $gray-alt: #dcdcdc;
 $gray-alt2: #ddd;
 $gray-alt3: #e9ecef;
+$white: #fff;

```

Comment:
Can we use the Bootstrap colors in some cases here? For example for `$white` and `$black`?

https://getbootstrap.com/docs/5.3/customize/color/#all-colors

Reply:
Maybe I don't know how to access them properly, but when I try to use them without declaring them first I get error "Error: Undefined variable." I think because we declare our variables in this file _before_ we import the bootstrap ones.